api/Document.json

```diff
@@ -5988,10 +6006,12 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": false
+              "version_added": "1",
+              "version_removed": true
```

Comment:
Can you find out and fill in the removal version? I've checked that `document.height` wasn't in Firefox 40, which was 5 years ago, so the original state of this entry was more useful in practice than this, even if it failed to note that this API was once supported.

More generally it would be good to not introduce new trues, unless finding the real version is too hard.

Reply:
I planned to fill those versions out but I guess I forgot to, whoops!